.yarn/versions/7b489a47.yml

```
@@ -0,0 +1,2 @@
+releases:
+  "@yarnpkg/plugin-workspace-tools": patch
```

```suggestion
  "@yarnpkg/plugin-workspace-tools": minor
```

A `feat` is a `minor` version bump